### PR TITLE
[EGE-482] feat(st-toggle-buttons): Update styles

### PR DESCRIPTION
**Pull request by paleu-stratio**
### Documentation
- [ ] Add the issue in PR description
- [ ] Have changelog updated
- [ ] You fill the milestone value
- [ ] You add some label
- [ ] Have example running in demo app
- [ ] Review id on demo is the same of egeo folder

### Code
- [ ] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [ ] Have at least 70% tests coverage

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,6 +68,10 @@
 * st-search: Allow to differentiate search events by its origin
 * st-dropdown-menu: Allow to display texts with HTML tags 
 
+**New features:**
```
Comment (emgaitan-stratio) [inline CHANGELOG.md]: Put this line in changelog of 16.4.0

Comment (paleu-stratio) [reply CHANGELOG.md]: Done!

File: src/lib/st-toggle-buttons/st-toggle-buttons.component.html [hunk 2]
```diff
@@ -10,15 +10,18 @@
...
-      <span *ngFor="let tab of tabs; let i = index;" (click)="onClick(tab)" class="st-toggle-buttons__tab sth-toggle-buttons__tab" [attr.id]="qaTag + '-' + tab.label"
-         [ngClass]="{'st-toggle-buttons__tab--active': tab.active, 'sth-toggle-buttons__tab--active': tab.active}">
-      {{tab.label}}
-      <span *ngIf="tab.number?.toFixed">({{tab.number}})</span>
-      </span>
-   </div>
+   <ul class="st-toggle-buttons__buttons sth-toggle-buttons__buttons">
+      <li *ngFor="let tab of tabs; let i = index;"
+         (click)="onClick(tab)">
+         <button [attr.id]="qaTag + '-' + tab.label" class="st-toggle-buttons__button sth-toggle-buttons__button"
```
[the remark this answers is not in the document]
Comment (fjsc) [reply src/lib/st-toggle-buttons/st-toggle-buttons.component.html]: example: 

cleanedId (id: string): string {
   const id = "Invalid id is [] {} - &";
   const cleanedId = id.replace(/\W/g,'_');
   return cleanedId;
}

[the remark this answers is not in the document]
Comment (paleu-stratio) [reply src/lib/st-toggle-buttons/st-toggle-buttons.component.html]: Done!
image

**Pull request merged**